Jenkinsfile

```diff
@@ -0,0 +1,263 @@
+                sh """
+                python3.7 contrib/intel/jenkins/build.py --builditem='fabtests'
+                python3.7 contrib/intel/jenkins/build.py --builditem='fabtests' --build_mode='dbg'
+                python3.7 contrib/intel/jenkins/build.py --builditem='fabtests' --build_mode='dl'              
+                echo 'fabtests build completed' 
+                """
+                }
+            }
+        }
+        
+        stage ('build-shmem') {
+            steps {
+              withEnv(['PATH+EXTRA=/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/bin']) {
+                sh """
+                python3.7  contrib/intel/jenkins/build.py --other_benchmarks='shmem'
+                echo 'shmem benchmarks built successfully'
+                """
+                }
+              }
+          }
+  
+        stage ('build OMPI_bm') {
+              steps {
+              withEnv(['PATH+EXTRA=/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/bin']) {
+                  sh """
+                  python3.7 contrib/intel/jenkins/build.py --mpi='ompi' 
+                  python3.7 contrib/intel/jenkins/build.py --mpi='ompi' --build_mode='dbg' 
+                  python3.7 contrib/intel/jenkins/build.py --mpi='ompi' --build_mode='dl'
+                  echo 'mpi benchmarks with ompi - built successfully'
+                 """
+                }
+              }
+          }
+    
+    stage('build IMPI_bm') {
+        steps {
+          withEnv(['PATH+EXTRA=/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/bin']) {
+                sh """
+                python3.7 contrib/intel/jenkins/build.py --mpi='impi'
+                python3.7 contrib/intel/jenkins/build.py --mpi='impi' --build_mode='dbg'
+                python3.7 contrib/intel/jenkins/build.py --mpi='impi' --build_mode='dl'
+                echo 'mpi benchmarks with impi - built successfully'
+                """
+            }
+          }
+      }  
+    
+    stage('build MPICH_bm') {
+        steps {
+          withEnv(['PATH+EXTRA=/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/bin']) {
+                sh """
+                python3.7 contrib/intel/jenkins/build.py --mpi='mpich'
+                python3.7 contrib/intel/jenkins/build.py --mpi='mpich' --build_mode='dbg'
+                python3.7 contrib/intel/jenkins/build.py --mpi='mpich' --build_mode='dl'
+                echo "mpi benchmarks with mpich - built successfully"
+                """
+              }
+            }
+        }
+   stage('parallel-tests') {
+            parallel {
+                stage('fab1-test') {
+                     agent {node {label 'FABRIC_1'}}
+                     steps{
+                        withEnv(['PATH+EXTRA=/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/bin/:$PYTHONPATH'])
+                        {
+                          sh """
+                            env
+                            ${env.ofi_install_dir}/${env.BRANCH_NAME}/${env.BUILD_NUMBER}/reg/bin/fi_info -f ${env.FABRIC}
```

Comment:
` ${env.ofi_install_dir}/${env.BRANCH_NAME}/${env.BUILD_NUMBER}` seems to be repeated in multiple places. Can it be replaced by a variable?

Reply:
yes I have actually added code to push this into the runtest.py. so this statement will not be there. This is to be replaced by something like 
python3.7 runtest.py --basic_test=fi_info